### PR TITLE
login and registe works also posts

### PR DESCRIPTION
**Pull request by MaxTrgb**

### CODE REVIEW

File: my-app/src/Auth/Login.jsx [hunk 1]
```diff
@@ -14,20 +14,20 @@ const LoginSchema = Yup.object().shape({
...
 const submitHandler = async (values, formikBag) => {
+    
     const loginData = {
         name: values.name,
         password: values.password,
     };
 
     try {
-        const response = await fetch('http://localhost:80/auth/login', {
+        const response = await fetch('/auth/login', {
```
Comment (XardasCode) [inline my-app/src/Auth/Login.jsx:24]: Why below you are using /api/v1/ but here just /auth/login?

File: my-app/src/Games/postGrid.jsx [hunk 2]
```diff
@@ -65,26 +65,27 @@ const PostGrid = () => {
...
                         key={post.id}
                         id={post.id}
                         imgSrc={post.imgSrc}
                         title={post.title}
-                        genre={post.genre}
-                        description={post.description}
                         rating={post.rating}
+                        genre={post.genre.name}
+                        description={post.description}
+
```
Comment (XardasCode) [inline my-app/src/Games/postGrid.jsx:84]: remove empty line?

File: my-app/src/Games/postGrid.jsx [hunk 3]
```diff
@@ -65,26 +65,27 @@ const PostGrid = () => {
...
                     />
                 ))}
             </div>
-           
+
```
Comment (XardasCode) [inline my-app/src/Games/postGrid.jsx:88]: the same

File: my-app/src/Auth/Login.jsx (hunk 1 above)
Comment (MaxTrgb) [reply my-app/src/Auth/Login.jsx:24]: i`ll fix it

**Pull request merged**